install/README.md

```diff
@@ -18,6 +18,8 @@ Meta information for the GeoNet equipment network.
 * `dataloggers.csv` - Recording dataloggers
 * `connections.csv` - Datalogger and sensor connection details
 * `streams.csv` - Datalogger and recorder sampling configurations
+* `gains.csv` - Individual datalogger and sensor settings that may impact overall sensitivities
+* `calibrations.csv` - Individual sensor sensitivity values can be used rather than default values.
```

Comment:
does the "sensor sensitivity" need to go in both?
If I understand correctly we will use:
- calibrations: for sensor/datalogger sensitivities/responsivity
- gains: for site specific sensitivities/responsivity

If that is correct, I'd suggest to change this to something like
```
* `gains.csv` - site specific settings that may impact overall sensitivities
* `calibrations.csv` - Individual sensors and dataloggers sensitivity values that may impact overall sensitivities and shall be used rather than default values.
```